### PR TITLE
#37 Add windows test transport support

### PR DESCRIPTION
**Pull request by MrWinstead**
* Uses named pipes for new test gRPC servers and clients

### CODE REVIEW

**Comment by MrWinstead** [comment]
This is a fix for #37

File: lile.go [hunk 1]
```diff
@@ -111,33 +113,78 @@ func CreateServer() *grpc.Server {
 	return gs
 }
 
-// NewTestServer is a helper function to create a gRPC server on a unix socket
-// it returns the socket location and a func to call which starts the server
-func NewTestServer(s *grpc.Server) (string, func()) {
+//   Creates a server listener dependent on the underlying platform. Windows
+// hosts will have a Windows Named pipe, anything else gets a UNIX socket
+func getTestServerTransportOrPanic()(string, net.Listener) {
```
Comment (arbarlow) [inline lile.go]: I think this should just be named `getTestServerTransport` and then we can panic or do something appropriate in the new test server method?

Comment (MrWinstead) [reply lile.go]: Sure thing! I'll add it and whatever else needs to happen when this build finishes

**Comment by MrWinstead** [comment]
All requested changes should now be implemented. Please let me know if you'd like anything else changed.

**Comment by MrWinstead** [comment]
@arbarlow Is there anything left which I'll need to add to this addition?

**Pull request merged**